server/src/schema.js

```diff
@@ -24,10 +34,19 @@ type Mutation {
 }
 
 type Subscription {
-  taskCreated: Task,
-  taskModified: Task,
-  taskDeleted: ID
+  taskMutated: TaskMutationPayload!
 }
+
+type TaskMutationPayload {
```

Comment:
How about `type TaskSubscription`. Only because I have aversy to word `payload` used in code.